impeller/tools/impeller.gni

```diff
@@ -19,19 +19,12 @@ declare_args() {
 
   # Whether the OpenGLES backend is enabled.
   impeller_enable_opengles =
-      (is_linux || is_win || is_android) && target_os != "fuchsia"
+      (is_linux || is_win || is_android || is_mac) && target_os != "fuchsia"
 
   # Whether the Vulkan backend is enabled.
-  impeller_enable_vulkan = (is_linux || is_win || is_android ||
+  impeller_enable_vulkan = (is_linux || is_win || is_android || is_mac ||
```

Comment:
Why does this one have `enable_unittests` but the opengles one doesn't?

Reply:
Lets try it